src/components/Divider/Divider.tsx

```diff
@@ -29,16 +32,37 @@ const styleMap = {
 };
 
 export const Divider = ({
+    vertical = false,
     style = DividerStyle.Solid,
     height = DividerHeight.Small,
-    color: borderTopColor = '#CCC',
-}: DividerProps): ReactElement => (
-    <div className="tw-flex tw-items-center" style={{ height }} data-test-id="divider">
-        <hr
-            className={`tw-border-t tw-m-0 tw-w-full ${styleMap[style]}`}
-            style={{ borderTopColor }}
-            data-test-id="divider-hr"
-        />
-    </div>
-);
+    color = '#CCC',
+}: DividerProps): ReactElement => {
+    const verticalClassNames = `tw-w-px tw-h-full tw-border-r tw-m-0 ${styleMap[style]}`;
+    return (
+        <>
+            {vertical && (
+                <div
+                    className="tw-flex tw-self-stretch tw-mt-0 tw-mb-0 tw-items-center tw-justify-center"
+                    style={{ width: height }}
+                    data-test-id="divider"
+                >
+                    <div
+                        className={`tw-w-px ${verticalClassNames}`}
```

Comment:
`tw-w-px` is already present in `verticalClassNames` so you can just pass the variable -> `className={verticalClassNames}`